2023-10-05 Handbook/product#settled (#14270)
fleetdm/fleet: handbook/product/README.md

Once the draft has been approved, it moves to the "Settled" column on the drafting board. 

> Before assigning an engineering manager to [estimate](https://fleetdm.com/handbook/engineering#sprint-ceremonies) a user story, the product team should ensure the product section of the user story [checklist](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=story&projects=&template=story.md&title=) is complete. 

Reviewer: ```suggestion
Before assigning an engineering manager to [estimate](https://fleetdm.com/handbook/engineering#sprint-ceremonies) a user story, the product designer ensures the product section of the user story [checklist](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=story&projects=&template=story.md&title=) is complete. 
```

Reviewer 2: cc @ireedy

Reviewer 2: Always trying to reduce the surface area of the handbook if we can. We might not be able to here.

Author: @noahtalerman good point. I think we can remove the callout box now that we've specified the DRI within the section text. I'll update that now.

> For bugs that have gone through design and are considered "Settled", the designer should remove the `:product` label and move the issue to the 'To be scheduled' column on the bugs board for the attention of the product manager.

Reviewer: ```suggestion
Once a bug has gone through design and is considered "Settled", the designer removes the `:product` label and moves the issue to the 'To be scheduled' column on the "Bugs" board for the attention of the product manager.
```

What prevents this from going directly to the EM?

Reviewer 2: Unrelated to Mike's question above: When does a bug need to go through design?

Author: @mikermcneil the idea is for the bug to be routed through the PM first so it can be appropriately prioritized into the sprint backlog. This aims to avoid things slipping through or being prioritized incorrectly. I've updated the language to be more specific about this, so thanks for flagging that it wasn't clear.
